dart/lib/src/protocol/sentry_app.dart

```diff
@@ -51,6 +39,19 @@ class SentryApp {
   /// Application specific device identifier.
   final String? deviceAppHash;
 
+  /// Deserializes a [SentryApp] from JSON [Map].
+  factory SentryApp.fromJson(Map<String, dynamic> data) => SentryApp(
```

Comment:
a lot of methods were moved around and this make the review a bit harder as I don't know if they actually changed or not, I'd suggest formatting in a different PR, but for now, I'd just ask, all those movings around, is there something I should check or it was, just formatting?

Reply:
the comment is valid for all the `fromJson` methods

Reply:
It was just moving the fromJson methods directly above the toJson methods, thats the only case where i moved things around. The only other case was adding the "sentry_" prefix to test files where it was missing before, also for consistencies sake. But i will keep this in mind for future PRs, so that they are easier to parse for review. 🙇

Reply:
thanks